src/Compilers/CSharp/Portable/Symbols/Source/SourceLocalSymbol.cs

```diff
@@ -401,22 +401,24 @@ internal void SetTypeWithAnnotations(TypeWithAnnotations newType)
         /// Gets a new local symbol with the given TypeWithAnnotations as the new type. This
         /// type should be identical to the original except for nullability.
```

Comment:
>nullability [](start = 64, length = 11)

"... and tuple names"?

Reply:
It's not intended to be so, we just don't handle tuple names correctly in the walker currently.

---
In reply to: [327301886](https://github.com/dotnet/roslyn/pull/38769#discussion_r327301886) [](ancestors = 327301886)